eng/jacoco-test-coverage/pom.xml

```diff
@@ -115,42 +115,42 @@
     <dependency>
       <groupId>com.azure</groupId>
       <artifactId>azure-storage-common</artifactId>
-      <version>12.3.0</version> <!-- {x-version-update;com.azure:azure-storage-common;current} -->
+      <version>12.3.1</version> <!-- {x-version-update;com.azure:azure-storage-common;current} -->
     </dependency>
     <dependency>
       <groupId>com.azure</groupId>
       <artifactId>azure-storage-blob</artifactId>
-      <version>12.3.0</version> <!-- {x-version-update;com.azure:azure-storage-blob;current} -->
+      <version>12.3.1</version> <!-- {x-version-update;com.azure:azure-storage-blob;current} -->
     </dependency>
     <dependency>
       <groupId>com.azure</groupId>
       <artifactId>azure-storage-blob-batch</artifactId>
-      <version>12.3.0</version> <!-- {x-version-update;com.azure:azure-storage-blob-batch;current} -->
+      <version>12.3.1</version> <!-- {x-version-update;com.azure:azure-storage-blob-batch;current} -->
     </dependency>
     <dependency>
       <groupId>com.azure</groupId>
       <artifactId>azure-storage-blob-cryptography</artifactId>
-      <version>12.3.0</version> <!-- {x-version-update;com.azure:azure-storage-blob-cryptography;current} -->
+      <version>12.3.1</version> <!-- {x-version-update;com.azure:azure-storage-blob-cryptography;current} -->
     </dependency>
     <dependency>
       <groupId>com.azure</groupId>
       <artifactId>azure-storage-blob-nio</artifactId>
-      <version>12.0.0-beta.1</version> <!-- {x-version-update;com.azure:azure-storage-blob-nio;current} -->
+      <version>12.0.0-beta.2</version> <!-- {x-version-update;com.azure:azure-storage-blob-nio;current} -->
     </dependency>
```

Comment:
This hasn't been releasd, no need to rev versions.

Reply:
I just updated them but in the unified pipeline, we can skip releasing these if they don't have to be and this won't go to master anyway.